handler.go

```diff
@@ -169,6 +176,9 @@ func NewServerStreamHandler[Req, Res any](
 			if err != nil {
 				return err
 			}
+			ctx = newHandlerContext(ctx, &streamCallInfo{
+				conn: conn,
+			})
```

Comment:
I think we need to do this for client and bidi as well

Reply:
See above comment.